src/Toggles.ts

```diff
@@ -927,7 +927,9 @@ export const toggleAscStatPerSecond = (id: number) => {
     }
 
     el.textContent = player.ascStatToggles[id] ? '/s' : '';
-    el.textContent = player.ascStatToggles[6] ? '' : '';
+    if (id === 6){
+        el.textContent = player.ascStatToggles[id] ? '' : '';
```

Comment:
```suggestion
    if (id === 6) {
        el.textContent = '';
```

are you sure this is correct?

Reply:
![1](https://user-images.githubusercontent.com/78285417/178600917-6bb0b900-0dad-49f9-9cc1-70c42cef0a58.PNG)
![2](https://user-images.githubusercontent.com/78285417/178600925-39adb982-aa8f-4e51-9a40-a50f7c39582e.PNG)
both toggles